### DISCUSSION TITLE
Google 申請穿戴裝置專利，皮膚為操作介面

### DISCUSSION THREAD

**Discussion by Jay-722** [1101-GS4719-C/Python-Community-GS4719-C#478] [General]
據 Google 專利申請資料，包括無線耳機和手錶等穿戴裝置，可透過感測器和加速度感測器，偵測裝置附近用戶接觸皮膚的動作，因應產生的震動判斷點擊和滑動等操作，這項專利有助擴大操作面積，用戶毋須辛苦滑動點按小小的耳機或錶面。

這項新技術感測皮膚的動作聽起來很猛，但真的可以實行感覺還需要克服很多困難，畢竟身體會做許多動作，誤觸的情況也會很多。

文章:https://technews.tw/2022/03/11/google-pixel-watch-en-pixel-buds-met-skin-interface/

**Comment by sean1325418** [comment]
實體操作簡單，但背後隱藏著工程師的心血，想要解決那些誤觸防護聽起來簡單，但實際上感覺會寫程式寫到吐血。

**Comment by xkllkx** [comment]
感覺裝置如果採用此設計，使用者體驗會更佳的直覺。

**Comment by ThomasMeng0515** [comment]
這個裝置感覺有許多困難需要克服，像是皮膚假如出汗的話會不會影響到感測器的偵測，這些都需要被考量進去

**Comment by chupochun** [comment]
要如何解決誤觸的情況真的很難，光是我平時使用的耳機，每天都會不小心誤觸好幾次
